### DISCUSSION TITLE
3.0.117-1 example configuration missing

### DISCUSSION THREAD

**Discussion by eleaner** [moosefs/moosefs#563] [Q&A]
Hi

I recently installed version 3.0.117-1 and to my surprise the` /etc/mfs` folder is not even created.
Where I can find the example configuration files?

Thanks
Marcin

**Comment by pkonopelko** [comment]
Hi Marcin @eleaner,
I think Discussion #402 answers your question :)

Hope it helps,
Piotr

**Reply by eleaner** [reply]
ouch, I am kind of used to the examples popping into /etc/mfs :)
thank you
found them

**Reply by pkonopelko** [reply]
If you use official Debian packages, it is more likely you will find examples in `/usr/share/doc/${package_name}` than in `/etc`

**Reply by eleaner** [reply]
@pkonopelko 
do you still check your old mobile sometimes?
